exo-create/features/steps/steps-when.ls

```diff
@@ -10,7 +10,7 @@ require! {
 
 module.exports = ->
 
-  @When /^executing "([^"]*)"$/ (command, done) ->
+  @When /^executing "([^"]*)"$/, timeout: 20_000, (command, done) ->
```

Comment:
Hmm, strange. Let's keep an eye on that. What happens if you just run this test? Does it hang at some point?